PROME/Chapter3/src/Components/Movie.jsx

```diff
@@ -0,0 +1,42 @@
+import "../index.css";
+import { useState } from "react";
+function Movie(props) {
+  const [isHovering, setIsHovering] = useState(0);
+  return (
+    <span
+      className="wrapper"
+      onMouseOver={() => setIsHovering(1)}
+      onMouseOut={() => setIsHovering(0)}
+    >
+      <div className="overview">
+        {isHovering ? (
+          <div>
+            <div className="hovername">{props.name}</div>

```

Comment:
```suggestion
            <div className="hovername">{name}</div>

```
이렇게 props를 모두 없앨 수 있어요

Reply:
> 이렇게 props를 모두 없앨 수 있어요

참고하도록 하겠습니다!